index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <nav class="header">
+        <a href="#" class="logo">My Logo</a>
+        <img src="icons/menu-icon.svg" alt="menu-icon" class="menu-icon" />
```

Comment:
- [ ] Please wrap the image in a `button` element. This is because the hamburger is a button that shows the nav on mobile screens.

- [ ] Please wrap this button in a `nav` element. This is for semantic reasons.

Reply:
# Please wrap this button in a nav element. This is for semantic reasons.
- I need some explanation on that

Reply:
Hi @GutemaG, your header should look like this
`html
<header>
  <a logo />
  <nav>
    <button><img /> <button>
  </nav>
</header>
`.

This is best practice for HTML semantics.

Reply:
Thank you @Kamba56